docs/docs/guides/integrate/login-ui/typescript-repo.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Typescript Repository
+sidebar_label: Typescript Repository
+---
+
+To replace our current golang built login UI and showcase the use of our new resource and OIDC APIs, we've created the [Typescript Repository](https://github.com/zitadel/typescript).
+The Typescript Repository contains all TypeScript and JavaScript packages and applications you need to create your own ZITADEL Login UI.
+The repo implements the [OIDC standard](./oidc-standard)
+
+## Included Apps And Packages
+
+- `login`: The login UI used by ZITADEL Cloud, powered by Next.js
+- `@zitadel/server`: core components for establishing node client connection, grpc stub
+- `@zitadel/client`: core components for establishing web client connection, grpc stub
+- `@zitadel/react`: shared React utilities and components built with tailwindcss
+- `@zitadel/next`: shared Next.js utilities
```

Comment:
What utilities are in there?

Reply:
> What utilities are in there?

This is under heavy development and could still change. This will combine react components with a server side request wrapper for example Login with IDPs.

Reply:
Can we add what we currently have?